tests/test_cli.py

```diff
@@ -23,27 +23,23 @@ def test_cli_help_shows_help():
         .startswith("Usage: python -m httomo")
     )
 
-"""
-# TODO possibly re-enable after yaml checker is complete
+
 def test_cli_noargs_raises_error():
     cmd = [sys.executable, "-m", "httomo"]
     try:
         subprocess.check_output(cmd)
     except subprocess.CalledProcessError as e:
         assert e.returncode == 2
-"""
 
-"""
+
 def test_cli_check_pass_data_file(standard_loader, standard_data):
     cmd = [sys.executable, "-m", "httomo", "check", standard_loader, standard_data]
     check_data_str = (
         "Checking that the paths to the data and keys in the YAML_CONFIG file "
         "match the paths and keys in the input file (IN_DATA)..."
     )
     assert check_data_str in subprocess.check_output(cmd).decode().strip()
-"""
 
-"""
 def test_cli_pass_output_folder(
```

Comment:
Ah, in #306, the equivalent of `test_cli_pass_output_folder()` was added back in the form of `test_cli_pass_create_folder()`, I forgot that I could have uncommented this one instead... :sweat_smile:

Could you remove one to avoid duplicating the test? Sorry for the extra work, that's my fault for not using the commented-out one.